server/controllers/tasks/lib/build_task_docs.coffee

```diff
@@ -2,49 +2,82 @@ __ = require('config').universalPath
 _ = __.require 'builders', 'utils'
 { Promise } = __.require 'lib', 'promises'
 
-entities_ = __.require 'controllers', 'entities/lib/entities'
 searchEntityDuplicatesSuggestions = require './search_entity_duplicates_suggestions'
 { calculateRelationScore } = require './relation_score'
+getAuthorWorksData = require './get_author_works_data'
 hasWorksLabelsOccurrence = __.require 'controllers', 'entities/lib/has_works_labels_occurrence'
+{ turnIntoRedirection } = __.require 'controllers', 'entities/lib/merge_entities'
 { prefixifyInv } = __.require 'controllers', 'entities/lib/prefix'
+{ _id:reconcilerUserId } = __.require('couch', 'hard_coded_documents').users.reconciler
 
 module.exports = (entity)->
   Promise.all [
     searchEntityDuplicatesSuggestions entity
     getAuthorWorksData entity._id
   ]
-  .spread (suggestionEntities, authorWorksData)->
-    relationScore = calculateRelationScore suggestionEntities
-    Promise.all suggestionEntities.map(create(authorWorksData, relationScore))
-
-create = (authorWorksData, relationScore)->
-  { labels, langs } = authorWorksData
-  return (suggestionEntity)->
-    suggestionUri = suggestionEntity.uri
-    _.type suggestionUri, 'string'
-    hasWorksLabelsOccurrence suggestionUri, labels, langs
-    .then (hasOccurence)->
+  .spread (suggestions, authorWorksData)->
+    unless _.some suggestions then return []
```

Comment:
```suggestion
    unless suggestions.length > 0 then return []
```
no need to call [such a sophisticated function](https://github.com/lodash/lodash/blob/b97a9a3da01077337540a377bf87860c4d00db2c/collection/some.js#L56-L65) for such a simple test, no?